src/airport.js

```diff
@@ -1,6 +1,48 @@
+const Plane = require('./Plane');
+
 class Airport {
-  // here's a starting point for you
+  landedPlanes = [];
+  landedCapacity;
+
+  // constructor(defaultLandedCapacity) {
+  //   this.landedCapacity = defaultLandedCapacity;
+  // }
 
-}
+  constructor(defaultLandedCapacity) {
+    if (Number.isInteger(defaultLandedCapacity) && defaultLandedCapacity > 0) {
+      this.landedCapacity = defaultLandedCapacity;
+    } else {
+      this.landedCapacity = 0;
+    }
+  }
+  landPlane = plane => {
+    if (plane instanceof Plane && this.getLandedPlaneCount() < this.getLandedCapacity() && !this.isPlanePresent(plane)) {
+      this.landedPlanes.push(plane);
+      // return true;
+      //   } else {
+      //       return false;
+    }
+  }
+  getLandedCapacity() {
+    return this.landedCapacity;
+  }
+  newLandedCapacity(newCapacity) {
+    this.landedCapacity = newCapacity;
+  }
+  getLandedPlaneCount() {
+    return this.landedPlanes.length;
+  }
+  isPlanePresent(plane) {
+    const indexOfPlaneInLandedPlanes = this.landedPlanes.findIndex(landedPlane => landedPlane.id === plane.id);
+    return indexOfPlaneInLandedPlanes > -1;
+  }
+  takeOffPlane = plane => {
+    const indexOfPlaneInLandedPlanes = this.landedPlanes.findIndex(landedPlane => landedPlane.id === plane.id);
+    if (indexOfPlaneInLandedPlanes > -1) {
+      this.landedPlanes.splice(indexOfPlaneInLandedPlanes, 1);
+       } return (indexOfPlaneInLandedPlanes > -1) ? true : false;
+    }
```

Comment:
I could also have taken the approach here (re-takeOffPlane) used in the re-landing of Plane above. And not used a Boolean return but used a previous existing method such as getLandedPlaneCount